tests/integration/devfile/cmd_devfile_url_test.go

```diff
@@ -41,22 +45,21 @@ var _ = Describe("odo devfile url command tests", func() {
 	Context("Listing urls", func() {
 		It("should list url after push", func() {
 			var stdout string
-			componentName := helper.RandString(6)
 			url1 := helper.RandString(5)
 			host := helper.RandString(5) + ".com"
```

Comment:
Not an issue with your tests, but I just realized we're using a random URL for the ingress host, which would cause issues if we wanted to test that the URL actually exposed the app

Nothing for you to do, but something we need to keep in mind.

Reply:
yep, we need to provide a correct host when we're testing the ingress domain